### PR TITLE
Static Pages

### PR DESCRIPTION
**Pull request by phannh58**

### CODE REVIEW

**Comment by levanhuan** [comment]
ok

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,14 +1,20 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>BaseApp</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <title><%= t "title.home" %></title>
+  <%= stylesheet_link_tag    "application", media: "all", "data-turbolinks-track" => true %>
```
Comment (thangtx88) [inline app/views/layouts/application.html.erb]: tylesheet_link_tag    "application", media: "all", "data-turbolinks-track" :true

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+  <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
```
Comment (thangtx88) [inline app/views/layouts/application.html.erb]: <%= javascript_include_tag "application", "data-turbolinks-track" : true %>

File: Gemfile [hunk 3]
```diff
@@ -4,7 +4,6 @@ source 'https://rubygems.org'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.6'
```
Comment (nhunght) [inline Gemfile]: double quote

File: Gemfile [hunk 4]
```diff
@@ -4,7 +4,6 @@ source 'https://rubygems.org'
...
 # Use sqlite3 as the database for Active Record
-gem 'sqlite3'
 # Use SCSS for stylesheets
 gem 'sass-rails', '~> 5.0'
```
Comment (nhunght) [inline Gemfile]: double quote

File: Gemfile [hunk 5]
```diff
@@ -13,7 +12,7 @@ gem 'uglifier', '>= 1.3.0'
 gem 'coffee-rails', '~> 4.1.0'
```
Comment (nhunght) [inline Gemfile]: double quote

File: Gemfile [hunk 6]
```diff
@@ -13,7 +12,7 @@ gem 'uglifier', '>= 1.3.0'
...
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'therubyracer', platforms: :ruby
-
+gem 'bootstrap-sass','3.2.0.0'
```
Comment (nhunght) [inline Gemfile]: double quote

File: Gemfile [hunk 7]
```diff
@@ -13,7 +12,7 @@ gem 'uglifier', '>= 1.3.0'
...
 # Use jquery as the JavaScript library
 gem 'jquery-rails'
```
Comment (nhunght) [inline Gemfile]: double quote

File: Gemfile [hunk 8]
```diff
@@ -33,6 +32,7 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 # gem 'capistrano-rails', group: :development
 
 group :development, :test do
+  gem 'sqlite3'
```
Comment (nhunght) [inline Gemfile]: double quote

File: Gemfile [hunk 9]
```diff
@@ -33,6 +32,7 @@ gem 'sdoc', '~> 0.4.0', group: :doc
...
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug'
```
Comment (nhunght) [inline Gemfile]: double quote

File: app/views/static_pages/home.html.erb [hunk 10]
```diff
@@ -0,0 +1,8 @@
+<div class="center jumbotron">
+  <h1><%= t "static_pages.home_title" %></h1>
+  <h2><%= t "title.text" %> 
+    <a href="http://recruit.framgia.vn/"></a><%= t "title.title_name" %></h2>
+    <%= link_to t("signup.signup_button"), "#", class: "btn btn-lg btn-primary" %>
+    <%= link_to image_tag("fram2.jpg", alt: "Framgia logo"),
```
Comment (nhunght) [inline app/views/static_pages/home.html.erb]: i18n

File: config/database.yml.example [hunk 11]
```diff
@@ -1,25 +0,0 @@
...
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
-test:
-  <<: *default
-  database: db/test.sqlite3
-
-production:
-  <<: *default
-  database: db/production.sqlite3
```
Comment (nhunght) [inline config/database.yml.example]: đừng xóa file này

File: app/assets/stylesheets/custom.scss [hunk 12]
```diff
@@ -0,0 +1,145 @@
...
+
+/* footer */
+footer {
+  margin-top: 45px;
+  padding-top: 5px;
+  border-top: 1px solid #eaeaea;
+  color: light-gray;
+  a {
+  color: #555;
+     &:hover {
```
Comment (nhunght) [inline app/assets/stylesheets/custom.scss]: indent

File: app/assets/stylesheets/custom.scss [hunk 13]
```diff
@@ -0,0 +1,145 @@
...
+          color: #222;
```
Comment (nhunght) [inline app/assets/stylesheets/custom.scss]: indent

File: app/assets/stylesheets/custom.scss [hunk 14]
```diff
@@ -0,0 +1,145 @@
...
+           }
```
Comment (nhunght) [inline app/assets/stylesheets/custom.scss]: indent

File: app/assets/stylesheets/custom.scss [hunk 15]
```diff
@@ -0,0 +1,145 @@
...
+  }
+  small {
+        float: left;
```
Comment (nhunght) [inline app/assets/stylesheets/custom.scss]: indent

File: app/assets/stylesheets/custom.scss [hunk 16]
```diff
@@ -0,0 +1,145 @@
...
+  }
+  ul {
+      float: right;
```
Comment (nhunght) [inline app/assets/stylesheets/custom.scss]: indent

File: app/assets/stylesheets/custom.scss [hunk 17]
```diff
@@ -0,0 +1,145 @@
...
+      list-style: none;
+      li {
+         float: left;
```
Comment (nhunght) [inline app/assets/stylesheets/custom.scss]: indent

File: app/controllers/users_controller.rb [hunk 18]
```diff
@@ -0,0 +1,17 @@
+class UsersController < ApplicationController
+
+  def show
+  	@user = User.find(params[:id])
```
Comment (nhunght) [inline app/controllers/users_controller.rb]: indent + remove ()

File: app/controllers/users_controller.rb [hunk 19]
```diff
@@ -0,0 +1,17 @@
...
+  end
+
+  def new
+  	@user = User.new
```
Comment (nhunght) [inline app/controllers/users_controller.rb]: indent

File: app/controllers/users_controller.rb [hunk 20]
```diff
@@ -0,0 +1,17 @@
...
+  end
+
+  private
+    def user_params
+      params.require(:user).permit(:name, :email, :password,
```
Comment (nhunght) [inline app/controllers/users_controller.rb]: remove ()

File: app/controllers/users_controller.rb [hunk 21]
```diff
@@ -0,0 +1,17 @@
+class UsersController < ApplicationController
```
Comment (nhunght) [inline app/controllers/users_controller.rb]: em làm static pages mà sao lại có cả user controller nhỉ

**Pull request merged**